src/Parser/Shaman/Restoration/Modules/Features/AlwaysBeCasting.js

```diff
@@ -59,18 +59,6 @@ class AlwaysBeCasting extends CoreAlwaysBeCastingHealing {
     [SPELLS.EARTHBIND_TOTEM.id]: TOTEM_GCD, // Not a static GCD but 1 second - haste
   }
 
-  get nonHealingTimeSuggestionThresholds() {
-    const nonHealingTimePercentage = this.totalHealingTimeWasted / this.owner.fightDuration;
-    return {
-      actual: nonHealingTimePercentage,
-      isGreaterThan: {
-        minor: 0.3,
-        average: 0.4,
-        major: 0.45,
-      },
-      style: 'percentage',
-    };
-  }
   get downtimeSuggestionThresholds() {
```

Comment:
You can remove the downtimeSuggestionThreshold and the suggestions aswell since it is all inherited anyway.